lib/tasks/fix_db_params.rake

```diff
@@ -0,0 +1,30 @@
+class ConfigReport < Report
+  def metrics
+    self[:metrics]
+  end
+end
+
+class ForemanOpenscap::ArfReport < Report
```

Comment:
Does this blow if OpenScap is not installed?

Reply:
sorry, fat fingers push, this does not belong to this PR